CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release notes
 
+## Version 4.5 (05 Sept 2023)
+* Now works with Java 11 and Jenkins 2.176.4 or higher.
+
```

Comment:
We're not building a release with this PR, so let's remove the version number (you can use `X.X` if you want) and the date.

Also, let's rephrase:
> The plugin now requires Java 11, and Jenkins 2.176.4 or higher.

Reply:
Done https://github.com/jenkinsci/plasticscm-plugin/pull/71/commits/991d7bc1c0c4fb6384e5f7325096deb6f354bf24